velox/docs/stats.rst

```diff
@@ -0,0 +1,126 @@
+==============
+Stats Counters
+==============
+Stats counter is used to collect the metrics of important velox runtime events
+for monitoring purpose. The collected metrics can provide insights into the
+continuous availability and performance analysis of a velox runtime system. For
+instance, the collected data can help automatically generate alerts at an
+outage. Velox provide a framework to collect the customized stats which consists
+of three steps:
+
+**Stats Definition**: define the name, type and config parameters for each stats,
+and register through registerVeloxCounters() on velox runtime startup.
+
+**Stats Record**: when the event of a predefined stats happens in production, the
+velox runtime records the stats data point through the stats collection
+interface provided by BaseStatsReporter.
+
+**Stats Export**: BaseStatsReporter aggregates the collected data points based
```

Comment:
It would be nice to clarify what actions developers should take to export the counters.